snippets/measurement_of_interest_by_age_and_gender.ggplot

```diff
@@ -0,0 +1,17 @@
+# This plot assumes that most_recent_measurement_of_interest.sql has been run.
+options(repr.plot.height = 14, repr.plot.width = 8)
+
+measurement_of_interest_df %>%
+    filter(value_as_number < 9999999) %>% # Get rid of nonsensical outliers.
```

Comment:
@deflaux just food for thought, for these code snippets should we  consider adding additional data cleaning steps like removal of NANs prior to plotting?

Reply:
Agreed, good food for thought!!

When passed null data, conveniently ggplot2 and plotnine drop it and spit out a warning to bring to your attention that this has occurred.

Other needed cleaning will depend upon the particular measurement. So I added one cleaning step that should(?) generally be fine and hopefully that bit of code that prompts people to realize that they can add more logic right there if they want to change the plot further by omitting/modifying more of the data.

Reply:
Excellent - that addresses my basic question. Thanks!